### PR TITLE
Add data collector to the side menu. 

### PR DESCRIPTION
**Pull request by bmarika**

### CODE REVIEW

File: pages/contributionsList.Vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+<script>
+    export default {
+      name: 'Contributions-List',
+    }
+</script>
+
+<style scoped>
+
+</style>
```
Comment (davekago) [inline pages/contributionsList.Vue:13]: This file looks like a dup of `data_collector/ContributionsList.Vue`. Is it needed?

File: pages/contribute.Vue [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    export default {
+      name: 'Contribute',
+    }
+
+</script>
+
+
+<style scoped>
+
+</style>
```
Comment (davekago) [inline pages/contribute.Vue:15]: This file looks like a dup of `data_collector/Contribute.Vue`. Is it needed?

File: pages/contributionsList.Vue (hunk 1 above)
Comment (bmarika) [reply pages/contributionsList.Vue:13]: Let's discuss this offline

**Review by davekago — APPROVED** [review]
LGTM

**Pull request merged**